.env.example

```diff
@@ -23,6 +26,7 @@ DATABASE_NAME=accounts
 DATABASE_USER=accounts
 DATABASE_PASSWORD=accounts
 DATABASE_HOST=postgres
+DATABASE_PORT=5433
```

Comment:
With `5433` when starting the containers I get:

```
backend-1   | django.db.utils.OperationalError: connection failed: connection to server at "172.18.0.4", port 5433 failed: Connection refused
backend-1   | 	Is the server running on that host and accepting TCP/IP connections?
```

I set it to `5432` and then the containers startup fine and connect.

Reply:
Huh, so that's what the problem was. I expose it for 5433, but I guess 5432 is exposed on the docker network. Will remove that line, thanks!